vru_use_case_scripts/plot_sdsm_position_error

```diff
@@ -184,4 +192,4 @@ position_distances = calc_position_distances(detections_df, sdsm_df)
 for object_id, distance_data in position_distances.items():
     distances_np = np.asarray(distance_data).transpose()
     plot_positions(distances_np[0], distances_np[1], title=object_id)
-    plt.show()
+    plt.savefig(cli_args.plots_dir / f"{object_id}.png")
```

Comment:
If we only name these files with object_id will that conflict with other object specific plots we generate. Maybe we should name then {object_id}_sdsm_error.png